app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="app_name">Feeder</string>
+  <string name="navigation_drawer_open"/>
+  <string name="navigation_drawer_close"/>
+  <string name="action_settings">Настройки</string>
+  <string name="add_feed">Добавить ленту</string>
+  <string name="title_activity_edit_feed">Подписаться на ленту</string>
+  <string name="url">URL</string>
+  <string name="tag">Тег</string>
+  <string name="add_feed_search_hint">URL ленты</string>
+  <string name="title">Название</string>
+
+  <string name="save">Сохранить</string>
+  <string name="empty_feed_top">Здесь нечего читать. Хотите…</string>
+  <string name="empty_feed_open">
+ &lt;font color=#607D8B&gt;Открыть&lt;/font&gt; другую ленту?</string>
```

Comment:
These strings will show html code to the user.

Compare with the original: https://gitlab.com/spacecowboy/Feeder/blob/master/app/src/main/res/values/strings.xml#L31

```
<string name="empty_feed_open">
    <![CDATA[<font color="#607D8B">Open</font> another feed?]]></string>
```